client/state/data-layer/wpcom/sites/activity/from-api.js

```diff
@@ -29,6 +29,61 @@ export function transformer( apiResponse ) {
 	return map( orderedItems, processItem );
 }
 
+/**
+ * Description is an explanation of specific knowledge about an activity
+ *
+ * The description is a formatted explanation of an activity and may
+ * contain links to resources referenced in the explanation
+ *
+ * @param {Object} item from API response
+ * @returns {?Object} parsed notifications-formatted block
```

Comment:
MINOR: I believe this return type is inaccurate, `{?Object}` is a nullable object, whereas the return here appears to be `{(Object|undefined)}`

Reply:
ooh, good one

Reply:
+TypeScript | OCaml